backends/dart_spry/bin/dart_spry.dart

```diff
@@ -1,32 +1,37 @@
-import 'package:spry/json.dart';
-import 'package:spry/multer.dart';
-import 'package:spry/spry.dart';
-import 'package:spry_router/spry_router.dart';
+// ignore_for_file: depend_on_referenced_packages
```

Comment:
Why do you need this? Because of webfetch import?

I guess it should be in a way that it should work without this lint ignore.

Reply:
@SirusCodes Spry's FromData comes from `webfetch`, and form content via `formData.get()` has a `String | File | Blob | null` type signature. Dart does not support the Union type. Since you need the handler to return size, import it to determine whether it is a `Blob` type.

PS: The `FormData` implementation comes from MDN

Reply:
@SirusCodes If you want to handle FormData children data in the handler and don't want to add the `depend_on_referenced_packages` lint rule, the only way is to add the `webfetch` dependency in `pubspec.yaml`. But this is not necessary, since `spry` depends on `webfetch` and this rule has no side effects.

Reply:
How about exporting webfetch from spry?

Adding it to pubspec doesn't make sense and disabling this rule is something a lot of people would not want, also not sure but do IDEs directly import it or we have to write the import statement ourselves?

Reply:
@SirusCodes Not a good idea, Spry 3 designed the Responsible function, which will automatically adapt the return value and set the response body.

FormData is only a small part of the functionality supported in Spry, although the choice of `webfetch` is subjective. When I created `webfetch`, I defined it as **HTTP client**. Although Spry has plans to allow the use of `fetch`'s Response in the handler, I did not find a good solution for the implementation of FormData. On the contrary, MDN is very useful.

The main reason why I am unwilling to actively export `webfetch` is that File is defined in MDN. It has the same name as File in `dart:io`, which can easily lead to developers using it incorrectly.

In addition, I don’t want developers to actively rely on webfetch unless necessary, because the next function of Spry involves the `request.fetch` function, which allows developers to obtain the results of remote or other routes in the Spry Application.

Once I export `webfetch` then Spry will export Request/Response with MDN design, which is incompatible with the design of Spry 3. Spry is designed as an extension of `dart:io`, and the main functions of Spry 3 are around` dart:io` performs magic.

Reply:
In the future, if `dart:io` adds FormData, according to the design concept of Spry 3, `webfetch` will be abandoned without hesitation. The design concept of `webfetch` follows the MDN Web APIs specification, which is more in line with the js/ts style and is completely deviated from the Spry 3 design concept.

Reply:
Another subjective wish is that I do not expect developers to use FormData, and would rather developers choose the design style of GraphQL APIs, REST full APIs or any RPC.

Reply:
> How about exporting webfetch from spry?
> 
> Adding it to pubspec doesn't make sense and disabling this rule is something a lot of people would not want, also not sure but do IDEs directly import it or we have to write the import statement ourselves?

When you select FormData, it is exported from spry, but when you need to confirm that FormData children are of type `Blob` or `File`, the IDE will automatically import the `webfetch` package.

Reply:
In addition, the reason for adding this rule is to avoid the prompts of the Dart analyzer. It is okay not to use this rule. It will not report any errors. It is just to not see `webfetch` in `pubspec.yaml` in the IDE. But the prompt of `webfetch` was imported.

Reply:
<img width="1252" alt="image" src="https://github.com/SirusCodes/backend_benchmark/assets/5564821/837bddb5-f22d-4acc-a0f1-00e4ac48495c">
This is my subjective wish to create this PR, just delete this rule.

Just like when I was developing Spry, I used a lot of file names that are not recommended by Dart. I would add `//ignore_for_file: file_names` rules to these files to prevent the IDE from prompting me that the file names do not comply with `package: lints/recommended.yaml` definition

Reply:
Yeah it makes sense 👍 Thanks for contributing.